setup.py

```diff
@@ -310,7 +310,7 @@ def finalize_options(self):
 
     url='http://radimrehurek.com/gensim',
     download_url='http://pypi.python.org/pypi/gensim',
-    
+
```

Comment:
unrelated change, please remove it & add needed thing for new cython file (see other extensions in `setup.py`). Also, don't forget to update `MANIFEST.in` and add `.rst` file for your model (Sphinx)

Reply:
still here, why you resolve it?